cpp-ch/local-engine/Functions/SparkFunctionGetJsonObject.h

```diff
@@ -293,7 +313,11 @@ class FlattenJSONStringOnRequiredFunction : public DB::IFunction
         {
             if (!col_json_const)
             {
-                std::string_view json{reinterpret_cast<const char *>(&chars[offsets[i - 1]]), offsets[i] - offsets[i - 1] - 1};
+                const char * src_chars = reinterpret_cast<const char *>(&chars[offsets[i - 1]]);
+                const size_t chars_length = offsets[i] - offsets[i - 1] - 1;
+                char dst_chars[chars_length];
+                // std::string_view json{chars_data, chars_length};
+                std::string_view json = copyJsonStringExceptCtrlChars(dst_chars, src_chars, chars_length);
```

Comment:
May try to parse in the normal way first. when parse failed, add `copyJsonStringExceptCtrlChars` and parse again.
Apply `copyJsonStringExceptCtrlChars` on each row may have performace issue.

Reply:
done